cilium/tests/conftest.py

```diff
@@ -0,0 +1,77 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import mock
+import pytest
+
+from datadog_checks.dev.kube_port_forward import port_forward
+from datadog_checks.dev.terraform import terraform_run
+from datadog_checks.utils.common import get_docker_hostname
+
+from .common import AGENT_METRICS, OPERATOR_METRICS
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+HOST = get_docker_hostname()
+AGENT_PORT = '9090'
+OPERATOR_PORT = '6942'
+AGENT_URL = "http://{}:{}/metrics".format(HOST, AGENT_PORT)
+OPERATOR_URL = "http://{}:{}/metrics".format(HOST, OPERATOR_PORT)
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    with terraform_run(os.path.join(HERE, 'terraform')) as outputs:
+        kubeconfig = outputs['kubeconfig']['value']
+        agent_instance = {}
+        operator_instance = {}
+
+        with port_forward(kubeconfig, 'cilium', 'cilium-operator', 9090) as (ip, port):
+            agent_instance = {'agent_endpoint': 'http://{}:{}/metrics'.format(ip, port), 'metrics': AGENT_METRICS}
+
+            with port_forward(kubeconfig, 'cilium', 'cilium-operator', 6942) as (operator_ip, operator_port):
```

Comment:
To avoid nesting let's use https://github.com/DataDog/integrations-core/blob/c13bfbe08bc0930140f92e5082f1023a78f562ec/istio/tests/conftest.py#L14-L17